### PR TITLE
Exams

### PR DESCRIPTION
**Pull request by YanPetrov7**
Ян Петров IM-12

### CODE REVIEW

File: Tasks/between.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-        if (i === -1) {
-          return  '' } else { str =  str.substring(0 , i)
-            }}}
-  return str
-}
+const getValueBetween = (str, fromLeft, fromRight) => {
+  let index = str.indexOf(fromLeft);
+  if (index === -1) {
+    return '';
+  } else {
```
Comment (tshemsedinov) [inline Tasks/between.js:8]: Use return-early principle and we don't need else here

File: Tasks/between.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+    index += fromLeft.length;
+    str = str.substring(index);
+    index = str.indexOf(fromRight);
+    if (index === -1) {
+      return '';
+    } else {
+      str = str.substring(0, index);
+    }
```
Comment (tshemsedinov) [inline Tasks/between.js:16]: Ditto

File: Tasks/words.js [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
+const countWords = (str) => {
+  let wordsNum = 0;
+  let flag = false;
+  for (const element of str) {
+    if (!flag) {
+      if (element === ' ') {
+        flag = false;
+      } else {
+        flag = true;
+      }
```
Comment (tshemsedinov) [inline Tasks/words.js:13]: Try to optimize this

File: Tasks/between.js (hunk 1 above)
Comment (YanPetrov7) [reply Tasks/between.js]: I thinked it will change the code style, but okay.

File: Tasks/between.js (hunk 2 above)
Comment (YanPetrov7) [reply Tasks/between.js]: Ditto

File: Tasks/words.js (hunk 3 above)
Comment (YanPetrov7) [reply Tasks/words.js]: Okay, you can see the changings in a new commit.

**Pull request closed without merging**